Bowling/Player.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Game
+{
+    public class Player
+    {
+        private List<Frame> shotsPerFrame = new List<Frame>();
```

Comment:
I know here you just started the logic...but are you sure that you will want to make the Player class to depend on Frame?